griptape/drivers/web_search/base_web_search_driver.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+from attrs import field, define
+
+from griptape.artifacts import ListArtifact
+
+
+@define
+class BaseWebSearchDriver(ABC):
+    results_count: int = field(default=5, kw_only=True)
+    language: str = field(default="en", kw_only=True)
+    country: str = field(default="us", kw_only=True)
+
+    @abstractmethod
+    def search(self, query: str, **kwargs) -> ListArtifact: ...
```

Comment:
Wondering if we need `kwargs` here. I know we use this pattern in a couple other Drivers but we rarely seem to leverage it.

Reply:
this was sort of future-proofing for passing in dynamic values